content/influxdb/v2.1/notebooks/troubleshoot-notebooks.md

```diff
@@ -10,10 +10,13 @@ menu:
 ---
 
 ### No measurements appear in my bucket even though there's data in it.
-Try changing the time range. You might have measurements prior to the time range you selected. For example, if the selected time range is `Past 1h` and the last write happened 16 hours ago, you'd need to change the time range to `Past 24h` (or more) to see your data.
+
+Try changing the time range. You might have measurements prior to the time range you  selected. For example, if the selected time range is `Past 1h` and the last write happened 16 hours ago, you'd need to change the time range to `Past 24h` (or more) to see your data.
 
 ### "No bucket exists" error message appears.
-This error appears when the Buckets API endpoint returns a list of bucket and you don’t have access to that bucket. Verify that you have access permissions to the bucket that you're trying to query or write to.
 
-### Metric selector crashes.
+This error occurs when the [Buckets API endpoint](/influxdb/v2.1/api/#tag/Buckets) returns a list of bucket and you don’t have access to that bucket. Verify that you have access permissions to the bucket that you're trying to query or write to. 
```

Comment:
Does it make sense to include the /write and /query endpoints here...is that when folks would get this error? (I don't see the "no bucket exists" error anywhere under the buckets endpoint. (Reads like this may be related to a list buckets request.)

Would it be accurate to say:
 "...occurs when a read or write request is sent to a bucket that the requestor does not have permissions to access. To verify a requestor has permissions to access the specified bucket, see how to [list all users with member privileges for a bucket](/influxdb/cloud/api/#operation/GetBucketsIDMembers).

Reply:
I don't know what is the situation where you would get this error in a notebook. We show a list of buckets to select from, and that list will only include the buckets you have access to. If you start typing in a bucket that doesn't match anything in the list, you get a 'no bucket found' message which could be that the bucket doesn't exist, or the bucket does exist but you don't have access to the bucket. In either case, there is nothing you can do. It doesn't really make sense for this to be in a section about notebook errors.

Reply:
thank you, @barbaranelson removing. 

Other item to add to "Troubleshooting notebooks"? Feels spare with the one section

Reply:
@veereshhosakere @bckeene have you heard/ran notebooks issues that may be helpful to add to the "Troubleshooting notebooks" doc? Right now, we only have one section in this doc.